### ISSUE TITLE
missing configuration for attribute role at NgbModal

### ISSUE THREAD

**Issue by cutiepiepony** [ng-bootstrap/ng-bootstrap#4101]
### feature request:
Using NgbModal for inserting a dialog offers no option to configure the value of the **role** attribute and is default "dialog".
It would be important [considering WCAG](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_alertdialog_role) to be able to set the value to "alertdialog" in certain cases.

### Versions of Angular, ng-bootstrap and Bootstrap:

Angular: 11.2.9

ng-bootstrap: 9.1.0

Bootstrap: 4.6.0
